Tests/TestingApps/PurchaseTesterSwiftUI/Shared/Views/Offerings/PromoOfferDetailsView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  PromoOfferDetails.swift
+//  PurchaseTester
+//
+//  Created by Josh Holtz on 2/1/22.
+//
+
+import Foundation
+import SwiftUI
+import RevenueCat
+
+struct PromoOfferDetailsView: View {
+    let package: RevenueCat.Package
+    
+    @State var eligibility: [RevenueCat.StoreProductDiscount: RevenueCat.PromotionalOffer?]? = nil
```

Comment:
Do you need all these `RevenueCat.`s everywhere?

Reply:
Oh! This is because I also had a V3 version and so I was using `RevenueCat.Package` and `Purchases.Package` 🙃 I will remove!